src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-left: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star {
+  background-image: url(images/star.svg);
```

Comment:
Only `.stars__star` should have `background-image: url(images/star.svg);`
For other (active) stars you use `:nth-child` where you should set which star is active